Serialization/Serialization/Helper.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Xml.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Runtime.Serialization;
+
+namespace Serialization
+{
+    public static class Helper
+    {
+        private static XmlSerializer xmlFormatter;
+        private static BinaryFormatter binaryFormatter;
+
+        static Helper()
+        {
+            xmlFormatter = new XmlSerializer(typeof(Student));
+            binaryFormatter = new BinaryFormatter();
+            
+        }
+
+        public static void SerializeToXml(Student student, Stream stream)
+        {
+            using (stream)
+            {
+                xmlFormatter.Serialize(stream, student);
+            }
+        }
+        public static Student DeserializeXml(Stream stream)
+        {
+            var student = xmlFormatter.Deserialize(stream) as Student;
+
+            return student;
+        }
+        public static void SerializeToBinary(Student student, Stream stream)
+        {
+            using (stream)
```

Comment:
same problem here with this using

Reply:
Fixed in a2b8f77f25c975083ebee2c718ad445e5576ff02